README.rst

```diff
@@ -14,6 +14,11 @@ a set of API to interact with them.
 
 .. |travis| image:: https://travis-ci.org/mozilla/servicebook.svg?branch=master
     :target: https://travis-ci.org/mozilla/servicebook
+    :alt: Build Status
+    
+.. |pyup| image:: https://pyup.io/repos/github/mozilla/stubattribution-tests/shield.svg
+     :target: https://pyup.io/repos/github/mozilla/stubattribution-tests/
```

Comment:
This is referencing stubattribution-tests..

Reply:
🥚 on my 🤦‍♂️